### PR TITLE
Ledger + ethers on Sei tutorial

### PR DESCRIPTION
**Pull request by dssei**
## What is the purpose of the change?

Add new tutorial

## Describe the changes to the documentation

Using Ledger wallet with ethers library

## Notes

### CODE REVIEW

File: pages/dev-tutorials/ledger-ethers.mdx [hunk 1]
```diff
@@ -0,0 +1,120 @@
+import {Code} from '@mantine/core';
+import {CHAIN_IDS} from '@sei-js/registry';
+
+# Using Ledger wallet with ethers on Sei
+
+Prerequisites:
```
Comment (cordt-sei) [inline pages/dev-tutorials/ledger-ethers.mdx:3]: Should we mention the extra step of setting `udevrules` for Linux users? That is a weird additional step to make Ledger devices work there.

Comment (dssei) [reply pages/dev-tutorials/ledger-ethers.mdx:3]: @cordt-sei So linux users need to run this https://github.com/LedgerHQ/udev-rules?

Comment (cordt-sei) [reply pages/dev-tutorials/ledger-ethers.mdx:3]: Yes

Comment (dssei) [reply pages/dev-tutorials/ledger-ethers.mdx:3]: Nice, addressed

**Review by cordt-sei — APPROVED** [review]

**Pull request merged**